exercises/practice/spiral-matrix/src/SpiralMatrix.elm

```diff
@@ -0,0 +1,6 @@
+module SpiralMatrix exposing (spiralMatrix)
+
```

Comment:
Should we provide the 'Direction' type here? 

It's just a thought ...

Reply:
No, I don't think so, that was just my own way of solving it, there are infinitely many other ways to do it.
Also, it's not in the instructions so it would be confusing.